irohad/consensus/yac/storage/yac_vote_storage.hpp

```diff
@@ -81,6 +83,9 @@ namespace iroha {
        public:
         // --------| public api |--------
 
+        /// @param consistency_model - consensus consistency model (CFT, BFT).
+        YacVoteStorage(ConsistencyModel consistency_model);
```

Comment:
Why not parametrize it with `std::shared_ptr<SupermajorityChecker>`? This will allow simpler testing with `MockSupermajorityChecker`